### DISCUSSION TITLE
Failed to select a version for the requirment 'fltk = "^1.4.33"' when I upgrade version of fltk from 1.4.15 to 1.4.33

### DISCUSSION THREAD

**Discussion by xavierzhao** [fltk-rs/fltk-rs#1582] [Q&A] [answered]
candidate versions found which didn't match" "fltk"

**Comment by MoAlyousef** [comment]
What's in your Cargo.toml?
Did you try to `cargo clean` then rebuild your project?

**Reply by xavierzhao** [reply]
> What's in your Cargo.toml? Did you try to `cargo clean` then rebuild your project?
Here is my Cargo.toml, please see picture, thanks
![image](https://github.com/user-attachments/assets/26c0a270-6224-437f-9a70-87000af0dab3)

**Reply by MoAlyousef** [reply]
Did you try to cargo clean then rebuild your project?

**Reply by xavierzhao** [reply, marked as answer]
> Did you try to cargo clean then rebuild your project?
Sorry, it was my bad. This problem was solved by upgrading rustc because my version of rustc was too old.
